src/main/java/com/amazon/ion/impl/macro/SystemMacro.kt

```diff
@@ -26,9 +26,14 @@ enum class SystemMacro(
     IfSingle(-1, IF_SINGLE, listOf(zeroToManyTagged("stream"), zeroToManyTagged("true_branch"), zeroToManyTagged("false_branch"))),
     IfMulti(-1, IF_MULTI, listOf(zeroToManyTagged("stream"), zeroToManyTagged("true_branch"), zeroToManyTagged("false_branch"))),
 
+    // Unnameable, unaddressable macros used for the internals of certain other system macros
+    // TODO: See if we can move these somewhere else so that they are not visible
+    _Private_FlattenStruct(-1, _systemSymbol = null, listOf(zeroToManyTagged("structs"))),
+    _Private_MakeFieldNameAndValue(-1, _systemSymbol = null, listOf(exactlyOneTagged("fieldName"), exactlyOneTagged("value"))),
+
     // The real macros
-    None(0, NONE, emptyList()),
-    Values(1, VALUES, listOf(zeroToManyTagged("values"))),
+    Values(1, VALUES, listOf(zeroToManyTagged("values")), templateBody { variable(0) }),
+    None(0, NONE, emptyList(), templateBody { macro(Values) { expressionGroup { } } }),
```

Comment:
🗺️ I switched the order of these in the enum (not the macro IDs, though) so that I could define `none` in terms of `values`. When we hook up a hard coded implementation for `none`, we can switch them back.